tests/init-global/warn/ScalaCheck.check

```diff
@@ -0,0 +1,10 @@
+-- Warning: tests/init-global/warn/ScalaCheck.scala:16:9 ---------------------------------------------------------------
+16 |  object OptMinSuccess extends IntOpt: // warn
+   |         ^
+   |       Cyclic initialization: object OptMinSuccess -> object FirstParser -> object OptMinSuccess. Calling trace:
+   |       ├── object OptMinSuccess extends IntOpt: // warn	[ ScalaCheck.scala:16 ]
+   |       │   ^
+   |       ├── object FirstParser extends CmdLineParser:	[ ScalaCheck.scala:15 ]
+   |       │   ^
+   |       └── val opts = List(OptMinSuccess)	[ ScalaCheck.scala:21 ]
+   |                           ^^^^^^^^^^^^^
```

Comment:
Sorry that I introduced the problem. As a quick fix, you can change `List` to `Some` in the original test.

Reply:
With this change, it is indeed working locally with the scala2_library_tasty